doc/ngsi/orion/user_and_programmer_guide/creating_orion_connection.md

```diff
@@ -0,0 +1,31 @@
+#Creating an Orion connection
+Creating an Orion connection means binding to a certain FIWARE service among all the services handled by a real Orion endpoint, by means of a Keystone token that only an authenticated and authorized user in that FIWARE service is able to obtain.
+
+Creating this kind of connection is not mandatory, but highly recommended in terms of configuration reusing. If not used, most of the configuration details needed in the connection will have to be replicated in every Orion-related Talend component.
+
+Start by creating a new job if you did not create it yet:
+
+![Create a job](./images/create_job.png)
+
+In the palette, within the package <i>NGSI/Orion</i>, there is a Talend component, `tOrionConnection`, in charge of defining all the required details for an Orion connection:
+
+* Log4j log file.
+* Log4j level.
+* Http-like Orion service endopoint, typically running on TCP/1026 port.
```

Comment:
Fixed in PR https://github.com/telefonicaid/iotp-talend-connectors/pull/4/commits/91793586c59de32feacb932ee31347e94d71284d